### DISCUSSION TITLE
Update 148 broke the control for me[Fixed]

### DISCUSSION THREAD

**Discussion by JesusPwnsYa** [Rem0o/FanControl.Releases#1586] [General]
For me all the sensors and outputs still work, but it seems like the process doesn't have control over my fans anymore.
I've been using fancontrol for over half a year, so I kind of know how it works by now. And since I didn't change anything (No driver updates in recent days) besides the update, it can only be the update itself.(Since it only started happening after the update)

If I wasn't quite sure I wouldn't want to bother you.

So to explain my issue in more detail:
My case fans have been fixed to 50%+ with a flat curve.
The 50% on them doesn't change at all, since my temps are quite low compared to my curves.

But the case fans go up from 880 to 1100 just by a small short load on my cpu. 
![image](https://user-images.githubusercontent.com/76225719/222491075-6c22aa3e-d4a7-4f44-b605-9e2116092465.png)
![image](https://user-images.githubusercontent.com/76225719/222491155-3888b160-129e-480d-9c15-1c13cb3514e5.png)
![image](https://user-images.githubusercontent.com/76225719/222491194-b5066d3e-69ea-4afa-8027-4e4f2692bbad.png)
![image](https://user-images.githubusercontent.com/76225719/222491724-9af6675b-f413-459c-ba87-0abc4ef5a4c5.png)
(Current | Minimum | Maximum | Average RPM)

hopefully it's just a small bug.

**Comment by Rem0o** [comment]
Take your front fan control, put it in manual mode, and slide the slider arround. Does the rpm change at all? 

From here, it seems like your control -> speed pairing is wrong, so the RPM you are monitoring on the control card is not the proper one.

Try and run this and see if your pairing is proper: 
![image](https://user-images.githubusercontent.com/14118956/222497527-9bcfd36e-0d74-4fc0-9c2e-d9c583bd7615.png)

**Comment by JesusPwnsYa** [comment]
Hi, thanks for the reply.
Manual control does control the fan.
![image](https://user-images.githubusercontent.com/76225719/222500745-bb16cf79-4805-48e3-b65e-4d7d5ee152d2.png)
![image](https://user-images.githubusercontent.com/76225719/222501005-fee0435c-f789-455b-a917-66a5e305c8cc.png)
![image](https://user-images.githubusercontent.com/76225719/222501296-03f00375-94ca-4e4a-af89-48dd20cf9866.png)

The thing is, I don't have any other fan control software installed and the default fancurves are set up in the BIOS.

Before the fan control seemed to be forced, now not so much.

**Reply by Rem0o** [reply]
I noticed Front Fan "S".  How many fans are plugged into that header?

**Reply by JesusPwnsYa** [reply]
3 each.
But funnily enough forcing all my fans to manual mode once seems to have fixed my issue. 
😅
Hope I don't have to do this after each restart, since a restart didn't fix my issue.

**Reply by Rem0o** [reply]
My question is, what RPM is it reading when daisy chaining fans like this?

**Reply by Rem0o** [reply]
Are all fans the same?

**Reply by JesusPwnsYa** [reply]
Yes, they are all the same models. Of course there will be a bit of difference between them, but I'm sure that wouldn't be 33%.
Top fans are 2x Noiseblocker eLoop B14-3(Up to 1400 RPM) and front fans are 3x NB PK-2(Up to 1200 RPM).
(Yes I mistakingly said 3 each above.)

**Comment by JesusPwnsYa** [comment]
So setting manual control fixed it for me, restarts didn't. After rebooting today it still works fine. 🤷
